backend/serve.go

```diff
@@ -291,6 +321,21 @@ func Manage(pluginID string, serveOpts ServeOpts) error {
 		}
 	}()
 
+	if serveOpts.HandlerMiddlewares == nil {
+		serveOpts.HandlerMiddlewares = make([]HandlerMiddleware, 0)
+	}
+
+	middlewares := []HandlerMiddleware{
+		newTenantIDMiddleware(),
+		newContextualLoggerMiddleware(),
+		NewTracingMiddleware(tracing.DefaultTracer()),
+		NewMetricsMiddleware(prometheus.DefaultRegisterer, "grafana", false),
```

Comment:
I assume the switch to use `namespace = "grafana"` and `subsystem = "plugin"` is virtually the same as the pre-existing `namespace = "grafana_plugin"`

Reply:
Used external plugin in local grafana with these changes and browsed the `http://localhost:3000/metrics/plugins/<plugin id>` and seems to work as expected 🎉 

```
# HELP grafana_plugin_request_total The total amount of plugin requests
# TYPE grafana_plugin_request_total counter
grafana_plugin_request_total{endpoint="queryData",status="ok",status_source="plugin"} 2
```